### ISSUE TITLE
sec 11.11.1 (learning rate schedule) has a null effect

### ISSUE THREAD

**Issue by murphyk** [d2l-ai/d2l-en#1715]
In the pytorch code, the difference between a constant learning rate of 0.3 and square root schedluing is non existent, despite claims that the latter works better.
